From c3c93684288a230ebf63ca8e07cc420db07d5f89 Mon Sep 17 00:00:00 2001
From: Jitendra Lanka <jlanka@nvidia.com>
Date: Wed, 22 Mar 2023 11:39:55 -0400
Subject: [PATCH backport 6.1.42 81/85] UBUNTU: SAUCE: mlxbf-ptm: use 0444
 instead of S_IRUGO

BugLink: https://bugs.launchpad.net/bugs/2011738

As recommended by checkscript, change S_IRUGO to 0444

Signed-off-by: Jitendra Lanka <jlanka@nvidia.com>
Acked-by: Tim Gardner <tim.gardner@canonical.com>
Acked-by: Bartlomiej Zolnierkiewicz <bartlomiej.zolnierkiewicz@canonical.com>
Signed-off-by: Bartlomiej Zolnierkiewicz <bartlomiej.zolnierkiewicz@canonical.com>
---
 drivers/platform/mellanox/mlxbf-ptm.c | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/drivers/platform/mellanox/mlxbf-ptm.c b/drivers/platform/mellanox/mlxbf-ptm.c
index 79c3e2902070..aeb68dc42e32 100644
--- a/drivers/platform/mellanox/mlxbf-ptm.c
+++ b/drivers/platform/mellanox/mlxbf-ptm.c
@@ -154,27 +154,27 @@ static int __init mlxbf_ptm_init(void)
 	monitors = debugfs_create_dir("monitors", ptm_root);
 	status = debugfs_create_dir("status", monitors);
 
-	debugfs_create_file("vr0_power", S_IRUGO, status, NULL,
+	debugfs_create_file("vr0_power", 0444, status, NULL,
 			    &vr0_power_fops);
-	debugfs_create_file("vr1_power", S_IRUGO, status, NULL,
+	debugfs_create_file("vr1_power", 0444, status, NULL,
 			    &vr1_power_fops);
-	debugfs_create_file("total_power", S_IRUGO, status, NULL,
+	debugfs_create_file("total_power", 0444, status, NULL,
 			    &total_power_fops);
-	debugfs_create_file("ddr_temp", S_IRUGO, status,
+	debugfs_create_file("ddr_temp", 0444, status,
 			    NULL, &ddr_thld_fops);
-	debugfs_create_file("core_temp", S_IRUGO, status,
+	debugfs_create_file("core_temp", 0444, status,
 			    NULL, &core_temp_fops);
-	debugfs_create_file("power_throttling_event_count", S_IRUGO, status,
+	debugfs_create_file("power_throttling_event_count", 0444, status,
 			    NULL, &pwr_evt_counter_fops);
-	debugfs_create_file("thermal_throttling_event_count", S_IRUGO, status,
+	debugfs_create_file("thermal_throttling_event_count", 0444, status,
 			    NULL, &temp_evt_counter_fops);
-	debugfs_create_file("throttling_state", S_IRUGO, status,
+	debugfs_create_file("throttling_state", 0444, status,
 			    NULL, &throttling_state_fops);
-	debugfs_create_file("power_throttling_state", S_IRUGO, status,
+	debugfs_create_file("power_throttling_state", 0444, status,
 			    NULL, &pthrottling_state_fops);
-	debugfs_create_file("thermal_throttling_state", S_IRUGO, status,
+	debugfs_create_file("thermal_throttling_state", 0444, status,
 			    NULL, &tthrottling_state_fops);
-	debugfs_create_file("error_state", S_IRUGO, status,
+	debugfs_create_file("error_state", 0444, status,
 			    NULL, &error_status_fops);
 
 	return 0;
-- 
2.20.1

